lib/seattleflu/db/cli/command/consensus_genome.py

```diff
@@ -0,0 +1,37 @@
+"""
+Upload consensus genomes and summary statistics to the warehouse receiving area.
+
+Consensus genomes and summary statistics should be in newline-delimited JSON
+format that matches those generated by the assembly pipeline.
+"""
+import click
+import logging
+import json
+from seattleflu.db.cli import cli
+from seattleflu.db.session import DatabaseSession
+
+
+LOG = logging.getLogger(__name__)
+
+
+@cli.command("upload-consensus-genome", help=__doc__)
+@click.argument("consensus_genome-file",
```

Comment:
Naming nitpick: I would suggest changing this to `consensus-genome upload` to match the command naming conventions used for `clinical`, `longitudinal`, etc.